### ISSUE TITLE
Renaming 'View' in menu

### ISSUE THREAD

**Issue by cooperzoe** [OpenRefine/OpenRefine#6281]
If I want to hide or unhide a column, at the moment I'd need to first navigate to 'View,' in the first menu tier, then select 'Collapse' options or 'Expand' options in the second menu tier. 

I think it would be more intuitive to rename 'View' to 'Collapse/Expand.' The term 'View' is a bit vague and could mean many things (Zoom in / Zoom out, for example). For the sake of clarity, I'd change the language to match the functionality itself.

I'd also like to propose for discussion: what if we changed the language to Hide/Unhide instead of Collapse/Expand? I think that would be even clearer for new users of the tool.

Here's a screenshot for context:
![Screenshot 2024-01-04 at 11 05 47 AM](https://github.com/OpenRefine/OpenRefine/assets/52141007/447dc1b0-a84b-42fb-b568-ff9e2d4c6b72)

**Comment by thadguidry** [comment]
If column metadata ever becomes a thing (kinda on our roadmap in our issues...somewhere), the term "View" might be fitting, but then again, for toggling viewing column metadata, perhaps later with extra header views, then a new toggle button (maybe a small gear on the left of each column) would be preferrable that offers many "View" settings options per column.

**Comment by Abbe98** [comment]
I fully agree with the suggestion to rename the "View" menu especially as "View" menus in many other programs have the meaning of hiding various application level elements.

Regarding the wording for collapsing/expanding of columns I would suggest Hide/Show as that seems common in spreadsheet applications. "Unhide" to me sounds like it suggests that the user hid the column which might not necessarily be the case making "Show" a little bit more neutral.

**Comment by thadguidry** [comment]
"Hide/Show" translates well.

**Comment by tfmorris** [comment]
I like "Hide/Show" but Excel, Numbers and Google Sheets all seem to use the Hide/Unhide terminology, so Hide/Show isn't universal.

**Comment by zyadtaha** [comment]
Hi, 
I would love to work on this.
I agree with renaming 'View' to 'Hide/Unhide', 'Collapse' to 'Hide', and 'Expand' to 'Unhide'.
What do you say ? @wetneb @tfmorris @thadguidry @cooperzoe

**Comment by wetneb** [comment]
Let's mark this as a good first issue, with either "Hide/Unhide" or "Hide/Show", up to the implementer.

**Comment by PadyA0** [comment]
hi there @wetneb , I just did a PR after testing my changes. It is ready for review. Please let me know if there are any suggestions.
